source/tutorials/importing.rst

```diff
@@ -121,6 +124,89 @@ Importing data
      --input-path emp-paired-end-sequences \
      --output-path emp-paired-end-sequences.qza
 
+.. _`multiplexed barcode in seq import`:
+
+Multiplexed single-end FASTQ with barcode in sequence
```

Comment:
Since we technically support dual-index demuxing via the methods in this section, plural barcodes is fine:

```suggestion
Multiplexed single-end FASTQ with barcodes in sequence
```